hspec.cabal

```diff
@@ -32,7 +32,7 @@ description:      Hspec is a testing framework for Haskell. It is inspired by
                   The Hspec Manual is at <http://hspec.github.io/>.
 
 extra-source-files:
-    changelog
+    CHANGES.markdown
```

Comment:
@sol oh that reminds me, we would like your feedback on incorporating ideas from hpack upstream. If you find us on #hackage irc we could chat about it.